portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,53 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Chris Elliott</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Chris Elliott</h1>
+      <div id="profile-links">
+        <a class="profile-link" href="https://github.com/zpChris" target="_blank">GitHub</a>
+        <a class="profile-link" href="https://www.linkedin.com/in/chris-m-elliott/" target="_blank">LinkedIn</a>
+        <a class="profile-link" href="/blog.html">Blog</a>
+      </div>
+      <img src="images/chris_profile.JPG" width="200px"/>
+      <h2>Background</h2>
+      <p>
+          I am a student in the University of Pennsylvania's Management and Technology Class of 2023, studying Computer Science and Economics.
```

Comment:
At Google, we request that all lines be kept under 80 characters. One of the reasons for this is that if you look at the github ui, this line (and others) run off the edge of the box and require a slider. This makes it a lot more cumbersome to review.

We have handy linters that let us know when our code is too long, so we won't be super pedantic about this, but I'd love if you went back through and shortened a lot of these lines.

Reply:
Ok, I'll look into linters that can help me keep my lines under 80 characters.

Reply:
No need to worry about it too much. The official policy for HTML is that breaking long lines is optional: go/htmlcssstyle#HTML_Line-Wrapping.

We're not expecting you to use linters or anything, so it's more of a best effort.

Reply:
Ok, that makes sense. Also, just to let you know, I can't view the link `go/htmlcssstyle` (the style guide resources I've tried to access have unfortunately been blocked). (I asked my team, and Eshika cannot view this either.)

Reply:
oh bummer, that's lame. Okay, I'll quote the relevant text to you here then:

"Break long lines (optional).

While there is no column limit recommendation for HTML, you may consider wrapping long lines if it significantly improves readability.

When line-wrapping, each continuation line should be indented at least 4 additional spaces from the original line."

Thanks for letting me know that link doesn't work for you!